.github/workflows/lint.yml

```diff
@@ -126,12 +130,39 @@ jobs:
                      --known-failure platform/DeviceSafeQueue.h \
                      --known-failure platform/GLibTypeDeleter.h \
                      --known-failure platform/SingletonConfigurationManager.cpp \
+                     src \
+                  "
+                  ./scripts/run_in_build_env.sh "./scripts/tools/not_known_to_gn.py \
+                     -e py -e matter -e lark -e yaml -e cpp -e java -e kt -e jinja -e proto \
+                     --gn-extra scripts/py_matter_idl/files.gni \
+                     --skip-dir scripts/py_matter_idl/build \
+                     scripts/py_matter_idl \
+                  "
+                  ./scripts/run_in_build_env.sh "./scripts/tools/not_known_to_gn.py \
+                     -e py -e cpp -e cc -e c -e h -e hpp -e mm \
```

Comment:
should we include lark as well? we did find a missing one

Also wondering about java since we have test output files for java.

Reply:
proto as well it seems.

Reply:
and jinja ... oh boy, list is long. Maybe re-check all extensions again just to be sure?

Reply:
The controller code does not uses any matter, jinja or lark files. The IDL does use it, and all these extensions are added (but only to IDL check).